Library/Homebrew/service/service.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
```

Comment:
May want to call this file `services/service.rb` because the `service/service/*` naming feels a bit redundant. May also be you can move everything in `service/service/*` into `services/` or `service/`.